wremnants/datasets/datagroups.py

```diff
@@ -96,9 +96,9 @@ def filterGroups(self, filters):
         else:
             new_groupnames = list(filter(filters, self.groups.keys()))
 
-        diff = list(set(new_groupnames) - set(self.groups.keys()))
+        diff = list(set(self.groups.keys() - set(new_groupnames)))
```

Comment:
I think the first ```set``` is no needed and this can be written as ```diff = list(self.groups.keys() - set(new_groupnames))```

Reply:
ok, I can edit it (then the reason for swapping is that the larger list had to be on the left, otherwise the diff was always empty)